protos/escrow/escrow.proto

```diff
@@ -216,3 +225,78 @@ message SignedCancelContractResult {
   CancelContractResult result = 1;
   bytes escrow_signature = 2;
 }
+
+message PayoutAddress {
+  bytes to_address = 1;
+  google.protobuf.Timestamp signed_time = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true
+  ];
+}
+
+message SignedPayoutAddress {
+  PayoutAddress address = 1;
+  bytes signature = 2;
+}
+
+message Payment {
+  string contract_id = 1;
+  int64 escrow_id = 2;
+  bytes from_address = 3;
+  bytes to_address = 4;
+  int64 amount = 5;
+  int32 index = 6;
+  ledger.SignedChannelState channel_state = 7;
+}
+
+message PaymentResult {
+  repeated Payment payment = 1;
+  bytes escrow_address = 2;
+  google.protobuf.Timestamp escrow_signed_time = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true
+  ];
+}
+
+message SignedPaymentResult {
+  PaymentResult result = 1;
+  bytes escrow_signature = 2;
+}
+
+message PayoutRquest {
+  repeated Payment payment = 1;
+  bytes seller_address = 2;
+  google.protobuf.Timestamp seller_signed_time = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true
+  ];
+}
+
+message SignedPayoutRquest {
+  PayoutRquest request = 1;
+  bytes seller_signature = 2;
+}
+
+message PayoutItem {
+  string contract_id = 1;
+  int64 escrow_id = 2;
+  bytes from_address = 3;
+  bytes to_address = 4;
+  int64 amount = 5;
+  int32 index = 6;
+  bool paid = 7;
```

Comment:
Does payment return all historically paid? Or it wont be returned after contract expired?

Reply:
No, only return new payments for seller. If already paid, will not be returned.

Reply:
So the only time this bool is false is because a payout failed, right? should this also include a reason why it failed? helps seller troubleshoot

Reply:
yes, will add the reason.